.github/issue_template.md

```diff
@@ -9,7 +9,7 @@ For all bugs, please provide the following information:
 ## Expected behavior and actual behavior
 
 ## Steps to reproduce the problem
-<!-- Prefer using code snippets rather than a screenshot. Please include a full minimal reproduction if possible. -->
+<!-- Prefer using code snippets rather than a screenshot. Please include a (full minimal reproduction)[https://stackoverflow.com/help/minimal-reproducible-example] if possible. -->
```

Comment:
```suggestion
<!-- Prefer using code snippets rather than a screenshot. Please include a [full minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) if possible. -->
```